pkgs/ffigen/test/native_objc_test/deprecated_config.yaml

```diff
@@ -0,0 +1,41 @@
+name: DeprecatedTestObjCLibrary
+description: 'Tests API deprecation'
+language: objc
+output: 'deprecated_bindings.dart'
+exclude-all-by-default: true
+objc-min-target-version:
```

Comment:
Maybe we should have multiple runs on the same objc file, with different ios and macos versions to see that:

* APIs get indeed omitted
* We generate some comments if an API is not available in the whole iOS or MacOS min/max target range. (e.g. also when an API is introduced _after_ the min OS version asked for. Not only when it's removed before the max OS version.)

Reply:
Done